src/Hit/Utils.hs

```diff
@@ -0,0 +1,16 @@
+module Hit.Utils
+       ( padLeft
+       , padRight
+       , maxLenOn
+       ) where
+
+import qualified Data.Text as T
+
+padLeft :: Int -> Text -> Text
+padLeft n t = T.replicate (n - T.length t) " " <> t
+
+padRight :: Int -> Text -> Text
+padRight n t = t <> T.replicate (n - T.length t) " "
+
+maxLenOn :: Foldable f => (a -> Text) -> f a -> Int
+maxLenOn f = foldl' (\acc a -> max acc $ T.length $ f a) 0
```

Comment:
Since all these functions are top-level functions in a separate module, it would be nice to add some small docs to them with minimal code examples :slightly_smiling_face: This will help in the future a lot!